src/MCPClient/lib/clientScripts/parse_mets_to_db.py

```diff
@@ -66,7 +66,10 @@ def parse_files(root):
             if not format_version:
                 name = amdsec.findtext('.//premis:formatName', namespaces=ns.NSMAP)
                 print('Format name', name)
-                format_version = fpr_models.FormatVersion.active.get(description=name)
+                try:
+                    format_version = fpr_models.FormatVersion.active.get(description=name)
+                except:
```

Comment:
I thought about doing that, but I don't think we want to ignore other possible exceptions.  In fact, I am starting to think that it would be better to remove the entire if statement:

https://github.com/artefactual/archivematica/blob/52c1bbdd538e92fda8879cfc5a8a4cb0a10f7a77/src/MCPClient/lib/clientScripts/parse_mets_to_db.py#L66-L72

The purpose of this code is to make sure that the premis object for each file , in the original METS file, gets copied into the database.  I think trying to guess the match from the premis object in the METS, to the fpr_formatversion, based on matching the premis name to the format version description, is not guaranteed to be accurate, and so we probably should not even try.